src/sentry/search/events/fields.py

```diff
@@ -2385,9 +2385,25 @@ def __init__(self, dataset: Dataset, params: ParamsType):
                         alias,
                     ),
                 ),
+                SnQLFunction(
+                    "eps",
+                    snql_aggregate=lambda args, alias: Function(
+                        "divide", [Function("count", []), args["interval"]], alias
+                    ),
+                    optional_args=[IntervalDefault("interval", 1, None)],
+                    default_result_type="number",
+                ),
+                SnQLFunction(
+                    "epm",
+                    snql_aggregate=lambda args, alias: Function(
+                        "divide",
+                        [Function("count", []), Function("divide", [args["interval"], 60])],
+                        alias,
+                    ),
+                    optional_args=[IntervalDefault("interval", 1, None)],
+                    default_result_type="number",
+                ),
```

Comment:
Could you add the `tpm`&`tps` aliases as well? just setting them in the dictionary to map to epm&eps will suffice (see how we did it with `FUNCTION_ALIASES` before)

Reply:
Added!